model/gen-ai/registry.yaml

```diff
@@ -332,3 +332,57 @@ groups:
 
           Additional output format details may be recorded in the future in the
           `gen_ai.output.{type}.*` attributes.
+      - id: gen_ai.evaluation.name
+        stability: development
+        type: string
+        brief: The qualified name of the evaluator used to evaluate the GenAI response.
+        examples: ["Relevance", "IntentResolution"]
+        note: >
+          This attribute identifies the evaluation name.
+      - id: gen_ai.evaluation.score
+        stability: development
+        type: double
+        brief: The evaluation score for the GenAI response.
+        examples: [4.0]
+      - id: gen_ai.input.metadata
+        stability: development
+        type: string
+        brief: Metadata associated with the GenAI input.
+        examples: ['{\"requestId\": \"fab3ee5d-a3c6-4c47-b3de-901bf02fa045\"}']
+        note: >
+          The structure is specific to the implementation. If the metadata is structured, it
+          is RECOMMENDED to provide it in a structured form using language-specific API. It
+          can also be captured as a JSON string when structured API is not available. If
+          metadata properties contain any sensitive information such as prompts or
+          completions, corresponding properties MUST NOT be recorded by default. Instrumentations
+          MAY provide a way to override this behavior and record sensitive information in
+          the metadata if user explicitly allows it.
+      - id: gen_ai.output.metadata
+        type: string
+        stability: development
+        brief: >
+          Metadata associated with the GenAI output.
+        note: >
+          The structure is specific to the implementation. If the metadata is structured, it
+          is RECOMMENDED to provide it in a structured form using language-specific API. It
+          can also be captured as a JSON string when structured API is not available. If
+          metadata properties contain any sensitive information such as prompts or
+          completions, corresponding properties MUST NOT be recorded by default. Instrumentations
+          MAY provide a way to override this behavior and record sensitive information in
+          the metadata if user explicitly allows it.
+        examples: ['{\"Perplexity\": 1.335}']
+        requirement_level: recommended
+      - id: gen_ai.evaluation.reasoning
+        stability: development
+        type: string
```

Comment:
do we know for sure that string is and will be enough?

Reply:
Looking for suggestions on:
1. Keep it string until we find a need to support other cases.
2. Make it `any` now.
Open to both, if needed I can make this change.